test/unit/serialize.js

```diff
@@ -142,8 +142,12 @@ QUnit.test( "serialize()", function( assert ) {
 		"action=Test&radio2=on&check=on&hidden=&foo%5Bbar%5D=&name=name&search=search&email=dave%40jquery.com&number=43&select1=&select2=3&select3=1&select3=2&select5=3&T3=%3F%0D%0AZ&H1=x&H2=&PWD=&T1=&T2=YES&My%20Name=me&S1=abc&S3=YES&S4=",
 		"Multiple form serialization as query string" );
 
-	assert.equal( jQuery( "#form, #testForm :input" ).serialize(),
-		"action=Test&radio2=on&check=on&hidden=&foo%5Bbar%5D=&name=name&search=search&email=dave%40jquery.com&number=43&select1=&select2=3&select3=1&select3=2&select5=3&T3=%3F%0D%0AZ&H1=x&H2=&PWD=&T1=&T2=YES&My%20Name=me&S1=abc&S3=YES&S4=",
-		"Mixed form/input serialization as query string" );
+	if ( jQuery.find.compile ) {
+		assert.equal( jQuery( "#form, #testForm :input" ).serialize(),
```

Comment:
I think this test should execute regardless of selector engine: `#form, #testForm :input` → `#form, #testForm input, #testForm select, #testForm textarea, #testForm button`.